airbyte-webapp/src/components/connection/CatalogTree/next/BulkEditPanel.module.scss

```diff
@@ -0,0 +1,71 @@
+@use "../../../../scss/colors";
+@use "../../../../scss/variables";
+
+.headerCell {
+  margin-right: variables.$spacing-xl;
+  padding-right: 0 !important;
```

Comment:
These !important in the file are because of styled components?

Reply:
yes